.travis.yml

```diff
@@ -7,14 +7,18 @@ matrix:
           env:
               - ELECTRON_CACHE=$HOME/.cache/electron
               - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
-          script:
+          after_success:
               # Build the release package.
               # Draft a new release when a new version was pushed to master.
               - yarn build --publish ${DEPLOY_TYPE}
         - os: linux
           services: docker
           language: generic
-          script:
+          before_script:
```

Comment:
Don't we need this on OS X?

Reply:
Not according to the docs: https://github.com/electron/spectron/blob/master/README.md#on-travis-ci

Also in their code & example, which is tested on both Linux and OSX, it only specifies this for Linux. See https://github.com/electron/spectron/blob/master/script/travis-build.sh